### ISSUE TITLE
Make NAFLM a separate repository?

### ISSUE THREAD

**Issue by mfranchetti** [TheNAF/naflm#151]
At the moment, NAFLM is a fork of the original OBBLM code. However, I do not believe there are any plans to push the development work back to the original project.
Currently, GitHub does not support searching the code of forked repositories, only of the parent repository. This makes it harder to track dependent code within the NAFLM files, especially as they are increasingly different from the OBBLM files.
Is it worth considering moving NAFLM development to a new project to improve the code searching ability? I'm aware this will be heavily dependent on @williamleonard and his plans/preferences for the OBBLM project.

**Comment by hutchinsfairy** [comment]
Would we lose a history of the commits so far since OBBLM?

**Comment by mfranchetti** [comment]
No, I don't think so - this [help page](https://help.github.com/articles/why-are-my-contributions-not-showing-up-on-my-profile/) suggests that it can be done by GitHub support:

> To detach the fork and turn it into a standalone repository on GitHub, contact GitHub Support.

**Comment by Shteve0** [comment]
The idea has been to eventually, perhaps gradually, roll some or all of the
developments back into obblm. I believe Will is starting to look at that
with a view to using the latest version for his mercuryvps offerings?
